### ISSUE TITLE
A11y contrast: Lack of high contrast colors, the pastel color theme against white is hard to see.

### ISSUE THREAD

**Issue by tchalvak** [raimohanska/ourboard#174]
I note that there are only two colors that really work as high contrast A11y colors against the white background: red or black.

Using the other pastel colors on the white results in the below.
![image](https://user-images.githubusercontent.com/23353/137378360-838ee5aa-2bf5-4e22-88dd-3729a9d6a10a.png)
^ an attempt to set up multiple voting dots for usage, some of them become almost invisible due to contrast.

It's great that the pastel color scheme is so subtle, but it'd be great to have more high contrast colors as well.

**Comment by raimohanska** [comment]
More colors would be great for sure. To enable this, I think we need a "pop-up" style color selector instead of laying all the colors flat onto the context menu.

**Comment by raimohanska** [comment]
Now we have a separate pop-up for colors, so adding colors should be easy! Want to take a stab? Start at `colors.tsx`.
